### ISSUE TITLE
About the self-attention style weighted sum

### ISSUE THREAD

**Issue by JiyueWang** [DefangChen/OKDDip#3]
Are those learned weights really necessary? Have you tried a random weight sum?
It seems that a random weighted sum is sufficient for your purpose of 'asymmetric' and 'dynamic'.

**Comment by DefangChen** [comment]
Yes, several ablation studies for attention weights have been conducted. Please refer to our paper carefully.

**Comment by JiyueWang** [comment]
Have you tried other simpler parameterized methods like fully connected network or plain Attention methods? Or just apply attention to the logits or probabilities instead of intermediate features?
